transactions/1.3/_index.md

```diff
@@ -3,7 +3,9 @@ title: "Jakarta Transactions 1.3"
 date: 2019-09-10
 summary: "First release for Jakarta EE 8"
 ---
-Jakarta Transactions
+Jakarta Transactions.
+
+This specification is a re-release of [JSR 907](http://jcp.org/en/jsr/detail?id=907) under the [EFSL](https://www.eclipse.org/legal/efsl/).
```

Comment:
Maybe OK, but just a comment.
JTA 1.3 has not been released from JCP, nor included in Java EE 8.
JTA 1.2 is the last version released from JCP.

Reply:
> JTA 1.3 has not been released from JCP,

I'm confused: https://jcp.org/aboutJava/communityprocess/mrel/jsr907/index3.html

Reply:
Even more confused now:

Here
![image](https://github.com/user-attachments/assets/d0201f87-4e69-43cf-ae0f-97746139dc58)

named as MR2.

And on download page it's MR3:
![image](https://github.com/user-attachments/assets/ea76a412-af99-495f-8e62-9df2cf9b9227)

And the property in PDF:
![image](https://github.com/user-attachments/assets/25f04844-be0f-49cb-a5c0-f6341a0826d8)

Nice.

Reply:
Anyway, my comment, "JTA 1.3 has not been released from JCP", was incorrect.  Sorry to confusion.